Formula/lib/libepoxy.rb

```diff
@@ -28,18 +28,15 @@ class Libepoxy < Formula
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
-  depends_on "python@3.11" => :build
```

Comment:
why not do `uses_from_macos "python" => :build`

Reply:
Since it will unnecessarily install `python@3.11` on Linux.

`meson` is capable of using the already installed `python@3.12`.